src/types/HTTP/http.model.ts

```diff
@@ -0,0 +1,14 @@
+export enum HttpStatusCode {
+    OK = 200,
+    BAD_REQUEST = 400,
+    ALREADY_EXISTS = 403,
+    NOT_FOUND = 404,
+    INTERNAL_SERVER = 500,
```

Comment:
You should use one naming convention around enums - here you are using `MACRO_CASE` in next enum you are using `PascalCase`. For enums I prefer `PascalCase` tbh.

Reply:
ja jestem za wypierdoleniem tego w calosc i uzycie tego co podeslalem w linku do npm

Reply:
https://www.npmjs.com/package/http-status-codes